_data/social.yml

```
@@ -104,7 +104,9 @@ websites:
     img: hootsuite.png
     tfa:
       - totp
+      - email
     doc: https://help.hootsuite.com/hc/en-us/articles/204595950
+    exception: Email is always enabled as backup method for Software Token.
```

```suggestion
    exception: "Email is always enabled as backup method for Software Token."
```
Quotes are not strictly necessary, but used by convention.